check_stability.py

```diff
@@ -386,24 +387,25 @@ def get_files_changed():
 
 def get_affected_testfiles(files_changed):
     affected_testfiles = []
-    ignore_dirs = ["conformance-checkers", "docs"]
-    manifest_file = os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests", "MANIFEST.json")
-    with open(manifest_file, "r") as fh:
-        manifest_contents = fh.read()
-    for changedfile_pathname in files_changed:
-        repo_root = os.path.abspath(os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests"))
-        changed_file_repo_path = changedfile_pathname[len(repo_root):]
-        if changed_file_repo_path in manifest_contents:
-            # This changed file is actually a test, so skip it.
-            continue
+    all_tests = []
+    nontests_changed = set(files_changed)
+    repo_root = os.path.abspath(os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests"))
+    manifest_file = os.path.join(repo_root, "MANIFEST.json")
+    for test,_ in manifest.load(repo_root, manifest_file):
+        all_tests.append(os.path.join(repo_root, test))
+        if test in nontests_changed:
```

Comment:
I think `test` here is a relative path and `nontests_changed` is absolute paths. Am I missing something or should you convert test to an absolute path as in `all_tests.append` above?